### PR TITLE
style: adapt the code following programming rules

### PR DESCRIPTION
**Pull request by PaablooCH**

### CODE REVIEW

**Comment by PaablooCH** [comment]
Task linked: [CU-865bph92u Coding style: correct cases, 120 column limit, etc.](https://app.clickup.com/t/865bph92u)

File: Source/DataModels/Resources/Resource.h [hunk 1]
```diff
@@ -58,20 +61,23 @@ class Resource
...
+	return changed;
 }
 
 inline void Resource::SetChanged(bool changed)
 {
 	this->changed = changed;
 }
 
-inline Resource::Resource(UID resourceUID, const std::string& fileName, const std::string& assetsPath, const std::string& libraryPath):
+inline Resource::Resource(	UID resourceUID,
```
Comment (markmeliaa) [inline Source/DataModels/Resources/Resource.h:77]: I'd improve the indentation here, it looks wacky.

**Review by markmeliaa — APPROVED** [review]
Great Pablo, thanks for your work!

File: Source/DataModels/Resources/Resource.h (hunk 1 above)
Comment (markmeliaa) [reply Source/DataModels/Resources/Resource.h:77]: Oh yea this is a constructor, you're right!

**Review by gerard-queralt — APPROVED** [review]
Thank you for your work, Pablo!

I'm not sure about removing all uses of `this`, as I think that it makes some methods easier to understand, but we can discuss this all together later and update the programming rules accordingly.

**Pull request merged**